how-to/what-is-vm-cache-and-how-do-I-use-it.md

```diff
@@ -0,0 +1,45 @@
+# What Is VMCache and How To Enable It
+
+* [What is VMCache](#what-is-vmcache-and-how-to-enable-it)
+* [How is this different to VM templating](#how-is-this-different-to-vm-templating)
+* [How to enable VMCache](#how-to-enable-vmcache)
+* [Limitations](#limitations)
+
+### What is VMCache
+VMCache is a new function that creates VMs as caches before using it.
+It helps speed up new container creation.  
+The function consists of a server and some clients communicating
+through Unix socket.  The protocol is gRPC in [protocols/cache/cache.proto](https://github.com/kata-containers/runtime/blob/master/protocols/cache/cache.proto).  
+The VMCache server will create some VMs and cache them by factory cache.
+It will convert the VM to gRPC format and transport it when gets
+requestion from clients.  
+Factory grpccache is the VMCache client.  It will request gRPC format
+VM and convert it back to a VM.  If VMCache function is enabled,
+kata-runtime will request VM from factory grpccache when it creates
+a new sandbox.
+
+### How is this different to VM templating
+Both [VM templating](https://github.com/kata-containers/documentation/blob/master/how-to/what-is-vm-templating-and-how-do-I-use-it.md) and VMCache help speed up new container creation.  
+When VM templating enabled, new VMs are created by cloning from a pre-created template VM, and they will share the same initramfs, kernel and agent memory in readonly mode.  So it saves a lot of memory if there are many Kata Containers running on the same host.  
+VMCache is not vulnerable to share memory CVE because each VM doesn't share the memory.
```

Comment:
Can you link the CVE here? Or at least the exact number.

Reply:
> Can you link the CVE here? Or at least the exact number.

Fixed.